DEVELOPER.md

```diff
@@ -23,13 +23,14 @@ Current availability versions are:
 
 ```
 furyctl
-├── install : looks for a Furyfile (default `./Furyfile.yaml`) and download it's content`
+├── init : Downloads Furyfile.yml and kustomization.yaml from the distribution repository to the current directory.
+├── vendor : looks for a Furyfile (default `./Furyfile.yaml`) and download it's content`
```

Comment:
`download it's content` -> `downloads its content`
But I'd just make it more specific than that: `downloads specified Fury modules`

I also suggest rewriting it all to:
`downloads Fury modules specified inside Furyfile.yaml`